Gemfile

```diff
@@ -3,3 +3,5 @@ source 'http://rubygems.org'
 
 
 gemspec
+
+gem 'httparty'
```

Comment:
why is it added here? its already in the gemspec

Reply:
sry was leftover from movement of the movement of the exporter to separate repo^^